.github/workflows/ci.yml

```diff
@@ -78,7 +82,7 @@ jobs:
         env:
           DISPLAY: ":99"
         run: |
-          ./node_modules/protractor/bin/webdriver-manager update --gecko=false
+          ./node_modules/protractor/bin/webdriver-manager update --versions.chrome=${CHROME_VERSION} --gecko=false
```

Comment:
whoops our functional tests broke (https://github.com/SEED-platform/seed/runs/3195021786?check_suite_focus=true#step:11:43)

Fix chrome version for both the driver and chrome itself so we don't have this issue again